README.md

```diff
@@ -72,13 +76,31 @@ To set up a local copy of the OpenCue website:
     ```
     npm install
     ```
+  
+1. Install `AutoPrefixer` to parse the CSS files.
```

Comment:
When introducing a command line snippet instruction like this, end the sentences on lines 80 and 86 with a colon (like you did on line 98).

Reply:
Thank you, I have implemented the required changes.